lib/data/remote/endpoints/status_endpoints.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:dio/dio.dart';
+
+import '../../../constants.dart';
```

Comment:
**Suggestion:** Use absolute imports, they are less prone to error and also more readable.